### PR TITLE
Selectively run e2e tests for serving, eventing

### PR DESCRIPTION
**Pull request by navidshaikh**
## Proposed Changes
 - Run only serving specific e2e tests locally
  ```
  E2E_TAGS="serving" ./test/e2e-tests-local.sh
  ```

 - Run only eventing specific e2e tests locally
  ```
  E2E_TAGS="eventing" ./test/e2e-tests-local.sh
  ```

 - Run all e2e tests locally
  ```
  ./test/e2e-tests-local.sh
  ```

### CODE REVIEW

**Comment by navidshaikh** [comment]
CI is running go 1.13 now, ref: [logs](https://prow.knative.dev/view/gcs/knative-prow/pr-logs/pull/knative_client/547/pull-knative-client-build-tests/1204656793528045569#1:build-log.txt%3A1020).

File: go.mod [hunk 1]
```diff
@@ -32,3 +32,5 @@ require (
 // Fix for `[` in help messages and shell completion code
 // See https://github.com/spf13/cobra/pull/899
 replace github.com/spf13/cobra => github.com/chmouel/cobra v0.0.0-20191021105835-a78788917390
+
+go 1.13
```
Comment (rhuss) [inline go.mod]: Is this PR based on #535 ?

Comment (navidshaikh) [reply go.mod]: Yes, will resolve the conflicts at once after 535 merged.

Comment (navidshaikh) [reply go.mod]: Rebased.

**Comment by navidshaikh** [comment]
/test pull-knative-client-integration-tests-latest-release

boskos

**Pull request merged**